docs/guides/qiskit-serverless.mdx

```diff
@@ -1,232 +1,32 @@
 
-## Alternatively, one can automatically select a system according to most available:
-# from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
-# from qiskit_serverless_tools.selectors import IBMLeastBusyQPUSelector
-#
-# backend = IBMLeastBusyQPUSelector(service).get_backend(min_num_qubits=127)
-# pm = generate_preset_pass_manager(optimization_level=3, backend=backend)
-# target_circuit = pm.run(abstract_circuit)
-
-# Step 3: Execute the target circuit
-sampler = Sampler(mode=backend, options={"default_shots": 1024})
-job = sampler.run([target_circuit])
-result = job.result()
-    
-# Step 4: Postprocess the results
-print(f" > Counts: {result[0].data.meas.get_counts()}")
-
-# save results of program execution
-# note: saved items must be serializable
-save_result(result)
+```bash
+pip install qiskit_serverless
 ```
-  </TabItem>
-
-  <TabItem value="SamplerV1" label="Sampler (V1)">
-    ```python
-# source_files/my_qiskit_pattern_resource_management.py
-
-from qiskit_ibm_runtime import QiskitRuntimeService, Session, Sampler, Options
-from qiskit.circuit.random import random_circuit
-from qiskit_serverless_tools.selectors import IBMLeastNoisyQPUSelector
-
-service = QiskitRuntimeService()
-
-# Step 1: Map quantum circuits and operators
-abstract_circuit = random_circuit(
-    num_qubits=5, depth=4, measure=True, seed=1234
-)
-
-# Step 2: Optimize the circuit for quantum execution with automatically selected system
-selector = IBMLeastNoisyQPUSelector(
-    service, circuit=abstract_circuit, transpile_options={"optimization_level": 3}
-)
-backend = selector.get_backend(min_num_qubits=127)
-target_circuit = selector.optimized_circuit
-
-## Alternatively, one can automatically select a system according to most available:
-# from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
-# from qiskit_serverless_tools.selectors import IBMLeastBusyQPUSelector
-#
-# backend = IBMLeastBusyQPUSelector(service).get_backend(min_num_qubits=127)
-# pm = generate_preset_pass_manager(optimization_level=3, backend=backend)
-# target_circuit = pm.run(abstract_circuit)
-
-# Step 3: Execute the target circuit
-with Session(service, backend=backend) as session:
-    sampler = Sampler(
-        options=Options(
-            execution={"shots": 1024}, transpilation={"skip_transpilation": True}
-        )
-    )
-    result = sampler.run(target_circuit).result().quasi_dists[0]
-
-# Step 4: Postprocess the results
-print(result)
-
-# save results of program execution
-# note: saved items must be serializable
-save_result(result)
-```
-
-  </TabItem>
-</Tabs>
 
-After creating this pattern, you can deploy and run it remotely with Qiskit Serverless as described above.
+This package requires `python` 3.10+. For users and organizations who want to run Serverless on custom infrastructure, follow the [deploy Qiskit Serverless on custom cloud resources guide](https://qiskit.github.io/qiskit-serverless/).
```

Comment:
I don't see anything with that exact title - is it this? : https://qiskit.github.io/qiskit-serverless/deployment/cloud.html

Reply:
Yep!